src/main/groovy/com/ullink/testtools/elastic/TestExportTask.groovy

```diff
@@ -123,40 +136,58 @@ class TestExportTask extends Exec {
                     default:
                         throw new IllegalArgumentException("'type' attribute of type ${type.getClass()} is not supported")
                 }
+
                 String id = sha1Hashed(it.getClassname() + it.getName() + it.timestamp)
                 IndexRequest indexObj = new IndexRequest(index, typeFinal, id)
                 processor.add(indexObj.source(output, XContentType.JSON))
             }
         }
 
+        def InputJSONFile = getEnrichment().featureJsonParser('getFeatureJsonFile')
+        def InputJSON = getEnrichment().featureJsonParser('getFeatureJsonParseText')
+
+        def indexFeature = "feature-" + indexPrefix + new SimpleDateFormat(indexTimestampPattern).format(InputJSONFile.lastModified())
+        String typeFinal = "feature"
+
+        for (object in InputJSON) {
+            String id = sha1Hashed(object.toString())
+            IndexRequest indexObjFeature = new IndexRequest(indexFeature, typeFinal, id)
+            object.productName= getProperties().product.name
+            processor.add(indexObjFeature.source(object, XContentType.JSON))
+        }
+
         processor.close()
     }
 
     static def sha1Hashed(String value) {
         def messageDigest = MessageDigest.getInstance("SHA-1")
-        String hexString = messageDigest.digest(value.getBytes()).collect { String.format('%02x', it)}.join()
+        String hexString = messageDigest.digest(value.getBytes()).collect { String.format('%02x', it) }.join()
         return hexString
     }
 
     List<Result> parseTestFiles(List<File> files) {
         def list = []
         files.each {
             def xmlDoc = new XmlSlurper().parse(it)
+            def fileName = (xmlDoc.@name)
             String timestamp = xmlDoc.@timestamp
+            def solutions = getEnrichment().testCaseJsonParser(fileName)
+
+            def filePath = Paths.get("src", "test", "groovy", fileName.toString().replace(".", File.separator) + ".groovy")
 
             xmlDoc.children().each {
                 if (it.name() == "testcase") {
-                    Result result = parseTestCase(it)
+                    Result result = parseTestCase(it,solutions)
                     result.timestamp = timestamp
```

Comment:
you can remove the timestamp here, instead of done in TestExportTask

Reply:
We need the timestamp here as it is used to create the elastic search index later for each test case.